package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -55,7 +77,105 @@ def to_json(self) -> str:
         return json.dumps(session_dict)
 
     def save(self):
-        """Save the session store info on db ."""
-
+        """Save the session store info on db and uploads it
+        to the cloud if a remote cloud path is provided ."""
         with self._db_session_class.begin() as session:
```

Comment:
Here the `as session` should be `as tx` (from transaction).

Reply:
Since `db_session_class` is indeed a class, `db_session-class.b egin()` does produce a `session` and not a transaction. It's basically shorthand for `Session().begin()`. See https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.sessionmaker.begin